_articles/network.md

```diff
@@ -0,0 +1,313 @@
+---
+layout: article
+title: Basic Networking Diagnosis
+description: Instructions on how to help diagnose networking issues.
+keywords:
+  - Support
+  - networking
+  - command line
+  - diagnosis
+image: http://support.system76.com/images/system76.png
+hidden: false
+section: software-applications
+
+---
+
+Finding out why your system is having network issues can be a daunting task.  We'll explore some useful commands 
+
+[]{#anchor}A quick review of the TCP/IP model
+---------------------------------------------
+
+First, let's take a moment to review the fundamentals of the TCP/IP network model.
+
+![TCP/IP OSI model](/images/network/tcpip.png)
+
+The layers in the TCP/IP network model, in order, include:
+
+-   **Layer 5:** Application
+-   **Layer 4:** Transport
+-   **Layer 3:** Network/Internet
+-   **Layer 2:** Data Link
+-   **Layer 1:** Physical
+
+We will assume that you are familiar with this model, and will proceed by discussing ways to troubleshoot issues at stack Layers 1 through 4.
+Where to start troubleshooting is situation-dependent. For example, if you can SSH to a server, but the server can't connect to a MySQL database, the problem is unlikely to be the physical or data link layers
```

Comment:
Missing a period at the end of this sentence.

Reply:
ahh, 2 lines should've been joined.